docs/tutorial.rst

```diff
@@ -3,6 +3,113 @@
 ========
 Tutorial
 ========
+There are two main ways of accessing the resources of the ``stdpopsim`` package
+that will be detailed in this tutorial. The first is via the command line
+interface (CLI). This is useful if you want to do a straightforward run of the
+models in the ``stdpopsim`` :ref:`catalog <sec_catalog>`. The other way to
+access the ``stdpopsim`` resources is via the python API. This is a bit more
+complicated however it allows for more advanced tasks. This tutorial will walk
+through both ways of using the ``stdpopsim`` package as well as some more
+advanced tasks you may wish to do.
+
+***************************
+Using the ``stdpopsim`` CLI
```

Comment:
Consider changing this heading title to `The command-line-interface`. (In general, users might not know what the abbreviation means, especially if they've just been linked to this part of the page from somewhere else.) This title is going to appear in the sidebar, so best to keep it as short as possible.

Reply:
Done. Went with the hyphenation "command-line interface" as per wikipedia.